ossdbtoolsservice/language/metadata_executor.py

```diff
@@ -90,3 +90,11 @@ def functions(self) -> List[tuple]:
         use the lightweight metadata query as it'll have N queries for N functions otherwise
         """
         return [f for f in self.lightweight_metadata.functions()]
+
+    def users(self) -> List[tuple]:
+        """return list of users, used for MySQLCompleter"""
```

Comment:
we can remove mysql completer reference from comment

Reply:
Done